test/unit/model/reference/WebReference.test.ts

```diff
@@ -177,7 +177,7 @@ describe('WebReference', () => {
 
       const xmlString = xmlSerializer.serializeToString(reference.toXml());
       expect(xmlString)
-        .toBe(`<year iso-8601-date="year">year</year><ext-link ext-link-type="uri" xlink:href="extLink">extLink</ext-link><article-title>I am articleTitle text</article-title><source>I am source text</source><date-in-citation iso-8601-date="2011-10-10">October 10, 2011</date-in-citation>`);
+        .toBe(`<element-citation><year iso-8601-date="year">year</year><ext-link ext-link-type="uri" xlink:href="extLink">extLink</ext-link><article-title>I am articleTitle text</article-title><source>I am source text</source><date-in-citation iso-8601-date="2011-10-10">October 10, 2011</date-in-citation></element-citation>`);
```

Comment:
We need to add the `publication-type` to this (missed in my last review). see - https://github.com/elifesciences/XML-mapping/blob/7cfdc3919715cb35940ced2f9a6866182bad2698/elife-00666.xml#L1461